src/style/shared/dialog.less

```diff
@@ -190,10 +190,20 @@ dialog::backdrop {
     width: 100%;
 }
 
-.dialog-pointer(@left) {
+.pointerMixin (@atTop) when (@atTop) {
+    top: -0.9rem;
+    transform: rotate(45deg);
+}
+
+.pointerMixin (@atTop) when (default()) {
+    bottom: -0.9rem;
+    transform: rotate(-135deg);
+}
+
+.dialog-pointer(@left, @atTop) {
+    .pointerMixin(@atTop);
```

Comment:
And this is how you call a prepared mixin, adding the properties it calculates into the class.